content/docs/command-reference/push.md

```diff
@@ -88,13 +89,13 @@ to push.
   the entire existing commit history of the project.
 
 - `-d`, `--with-deps` - determines files to upload by tracking dependencies to
-  the target DVC-files (stages). If no `targets` are provided, this option is
+  the target `.dvc` files (stages). If no `targets` are provided, this option is
   ignored. By traversing all stage dependencies, DVC searches backward from the
   target stages in the corresponding pipelines. This means DVC will not push
   files referenced in later stages than the `targets`.
```

Comment:
This one is still outdated. Just copy it from pull as mentioned in https://github.com/iterative/dvc.org/pull/1408#discussion_r437133314 please (but change "upload" for "download").